### PR TITLE
Add Typescript Usage to the README

### PR DESCRIPTION
**Pull request by danibram**
Thanks for this awesome library @niftylettuce! And thanks to @ChrisEdgington for the guide to use with typescript.
I added the steps on the README. If there are anything wrong please tell me, my english is not perfect ^^
Thanks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -740,6 +740,48 @@ You should be able to pass data via `worker.workerData` (see [Custom Worker Opti
...
+```js
+function typescript_worker() {
+    const path = require('path')
+    require('ts-node').register()
+    const workerData = require('worker_threads').workerData
+    require(path.resolve(__dirname, workerData.__filename))
+}
+```
+
+Then use the the function as path (see [Using functions for jobs](#using-functions-for-jobs)), and pass the path to your ts job file:
```
Comment (Nisgrak) [inline README.md:758]: The CI are failing because the repeated "the"

File: README.md [hunk 2]
```diff
@@ -740,6 +740,48 @@ You should be able to pass data via `worker.workerData` (see [Custom Worker Opti
 
 Note that you cannot pass a built-in nor bound function.
 
+## Typescript Usage
+
+As is mentioned in this issue: https://github.com/breejs/bree/issues/24 and thanks to @ChrisEdgington.  You can use typescript workers as:
```
Comment (Nisgrak) [inline README.md:745]: And this link without ``.  This should did the trick

```md
As is mentioned in [#24](https://github.com/breejs/bree/issues/24) and thanks to @ChrisEdgington.
```

**Review by Nisgrak — CHANGES_REQUESTED** [review]
With this changes the CI should pass 😄

File: README.md (hunk 1 above)
Comment (danibram) [reply README.md]: ouch! Fixed

File: README.md (hunk 2 above)
Comment (danibram) [reply README.md]: Yes, much better

**Comment by niftylettuce** [comment]
Just ping me once this is ready for merge!

**Comment by Wazbat** [comment]
This is only for using typescript worker threads, correct? What about using the bree library itself with typescript, or is that not yet possible?

**Comment by Nisgrak** [comment]
We didn't have a `@types` or bundled types, I don't think that gonna be so hard make a simple types, I try to make something.

Create a `@types` package or bundle with the lib itself? @niftylettuce

**Comment by shadowgate15** [comment]
@Nisgrak That is what I have already done. It is already released on npm thanks to DefinitelyTyped.

@Wazbat This is for bree TypeScript bindings not for workers.

@niftylettuce  this is ready!

**Comment by shadowgate15** [comment]
@danibram  could you convert the commit messages to conventional commits. Then we will be good to go.

**Comment by danibram** [comment]
I amended the last commit, is this ok? @shadowgate15

**Pull request merged**